app.js

```diff
@@ -1,16 +1,35 @@
 const express = require("express")
 const app = express()
-const {getApi, getTopics} = require("./controllers/api.controller")
+const {getApi, getTopics, getArticleById} = require("./controllers/api.controller")
 
 app.get("/api", getApi)
 app.get("/api/topics", getTopics);
+app.get("/api/articles/:article_id", getArticleById)
 
 
-app.use((req, res, next) => {
-    res.status(404).send({ msg: "not found" });
+app.all("*",(req, res, ) => {
+   res.status(404).send({ msg: "not found" });
   });
+
   
-  app.use((err, req, res, next) => {
+/// POSTSQL ERROR
+app.use((err, req, res, next) => {
```

Comment:
Nice! Consider extracting this out to its own error handling file to keep this file easy to read :)